src/vtk_calculator.py

```diff
@@ -40,19 +40,21 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument("in_meshname", metavar="inputmesh", help="The mesh (VTK Unstructured Grid) used as input")
-    parser.add_argument("function", help="""The function to evalutate on the mesh.
+    parser.add_argument("--mesh", "-m", dest="in_meshname", help="The mesh (VTK Unstructured Grid) used as input")
+    parser.add_argument("--function", "-f", dest="function", help="""The function to evalutate on the mesh.
             Syntax is the same as used in the calculator object, coordinates are given as e.g.  'cos(x)+y'.""")
-    parser.add_argument("--out", "-o", dest="out_meshname", default=None, help="""The output meshname.
+    parser.add_argument("--output", "-o", dest="out_meshname", default=None, help="""The output meshname.
             Default is the same as for the input mesh""")
-    parser.add_argument("--tag", "-t", dest="tag", default="MyScalar", help="""The tag for output data.
-            Default is MyScalar""")
-    parser.add_argument("--intag", "-it", dest="intag", help="""The tag for input data.
-            Used in diff mode. If not given tag is used.""")
+    parser.add_argument("--data", "-d", dest="data", default="MyData", help="""The name of output data.
+            Default is MyData""")
+    parser.add_argument("--indata", "-ind", dest="indata", help="""The name of input data.
```

Comment:
@kursatyurt I would still prefer `diffdata` here instead of `indata`

Reply:
I changed it. I also changed some arguments as `required` hence users have to provide them otherwise it throws the help.

Reply:
Much better, thanks